website/docs/index.html.markdown

```diff
@@ -64,15 +64,15 @@ The other way is **statically** define all the credentials:
 
 ```hcl
 provider "helm" {
-    kubernetes {
-        host     = "https://104.196.242.174"
-        username = "ClusterMaster"
-        password = "MindTheGap"
-
-        client_certificate     = "${file("~/.kube/client-cert.pem")}"
-        client_key             = "${file("~/.kube/client-key.pem")}"
-        cluster_ca_certificate = "${file("~/.kube/cluster-ca-cert.pem")}"
-    }
+  kubernetes {
+    host     = "https://104.196.242.174"
+    username = "ClusterMaster"
+    password = "MindTheGap"
+
+    client_certificate     = "${file("~/.kube/client-cert.pem")}"
```

Comment:
If we're making it 0.12 we should not use `"${}"`

Reply:
fixed in 1e69c0cbe